doc/integration_test_report.rst

```diff
@@ -506,6 +509,24 @@ Traceability via intermediate items
     :nocaptions:
     :stats:
     :coveredintermediates:
+    :coverage: <79
+
+.. warning due to below item-matrix: coverage is not 100% due to child RQT not covered
```

Comment:
```suggestion
.. warning in below item-matrix: coverage is not 100% due to child RQT not covered
```

Reply:
Applied in c9f9916d8189981636ed802fe4b9d72f07f03cab